src/symbolize/gimli.rs

```diff
@@ -31,9 +31,12 @@ extern crate std as mystd;
 #[cfg(windows)]
 #[path = "gimli/mmap_windows.rs"]
 mod mmap;
-#[cfg(unix)]
+#[cfg(all(unix, not(target_env = "devkita64")))]
 #[path = "gimli/mmap_unix.rs"]
 mod mmap;
+#[cfg(target_env = "devkita64")]
+#[path = "gimli/mmap_fake.rs"]
+mod mmap;
```

Comment:
While you're here, mind reorganizing this to be in a `cfg_if!` to avoid the `#[cfg(not(...))]`?